sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -256,8 +256,19 @@ public abstract partial class Response : System.IDisposable
     public sealed partial class ResponseError
     {
         public ResponseError(string? code, string? message) { }
+        public ResponseError(string? code, string? message, string? target, Azure.ResponseInnerError? innerError = null, System.Collections.Generic.IReadOnlyList<Azure.ResponseError>? details = null) { }
         public string? Code { get { throw null; } }
+        public System.Collections.Generic.IReadOnlyList<Azure.ResponseError> Details { get { throw null; } }
+        public Azure.ResponseInnerError? InnerError { get { throw null; } }
         public string? Message { get { throw null; } }
+        public string? Target { get { throw null; } }
+        public override string ToString() { throw null; }
+    }
+    public sealed partial class ResponseInnerError
```

Comment:
Do we want a second error type in the public API?  How is it different from the first?

Reply:
This follows the REST API guidelines: https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#rest-error-response-body-structure

`innererror` has a different structure than the outer error, and `details` is the same structure as the outer error.